### PR TITLE
Fixing ID issue for PM

### PR DESCRIPTION
**Pull request by arturnt**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,9 +45,10 @@ module.exports = function (config, pageType, pageData) {
       fbq('track', 'ViewContent', {
         content_name: product[0].name, //product name -- should be product[0].name
         //content_category: 'Apparel & Accessories > Shoes', //product category
-        content_ids: _.map(product[0].variants, function(variant) { //array of product SKUs
-          return product[0].tagLine + "-" + variant.vendorSku;
-        }), 
+         content_ids: content_type === "product_group" ? product[0].tagLine :
+          _.map(product[0].variants, function (variant) { //array of product SKUs
```
Comment (rileylnapier) [inline index.js]: this assumes a global lodash being available... which moving forward shouldn't be assumed.

use native map.  

also, this app shouldn't import symphony-feu @arturnt @ryan-symphony

Comment (arturnt) [reply index.js]: Couple of things on this:

1. This is a forked version of this library. Changes should be made in the underlying library not in this CR. 
2. As far as PM is concerned they will be on AEM relatively soon - meaning we can drop this fork altogether.

Comment (rileylnapier) [reply index.js]: oh damn, i didn't see that.  whoops

Comment (arturnt) [reply index.js]: no worries, totally agree with the underlying feedback

**Pull request merged**